hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/config/PropertyNames.java

```diff
@@ -247,4 +247,6 @@ private PropertyNames() {
     public static final String STATS_SPEEDOMETER_HALF_LIFE_SECS = "stats.speedometerHalfLifeSecs";
     public static final String WORKFLOWS_ENABLED = "hedera.workflows.enabled";
     public static final String VIRTUALDATASOURCE_JASPERDB_TO_MERKLEDB = "virtualdatasource.jasperdbToMerkledb";
+    public static final String BLOCKLIST_ENABLED = "blocklist.enabled";
```

Comment:
Maybe not it's accounts
```suggestion
    public static final String ACCOUNTS_BLOCKLIST_ENABLED = "accounts.blocklist.enabled";
```
or
```suggestion
    public static final String ACCOUNTS_PK_BLOCKLIST_ENABLED = "accounts.pk.blocklist.enabled";
```

Reply:
Updated.